test/operators/second_quantization/test_fermionic_op.py

```diff
@@ -82,6 +107,13 @@ def test_init_multiple_digits(self):
         ]
         self.assertListEqual(actual.to_list(), desired)
 
+    @data(str2str, str2tuple, str2list)
+    def test_init_empty_str(self, pre_processing):
```

Comment:
I think if we allow this for the `FermionicOp` it should also work for the `SpinOp`